src/Subscriber/UnsupportedBranchSubscriber.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Subscriber;
+
+use App\Api\Issue\IssueApi;
+use App\Event\GitHubEvent;
+use App\GitHubEvents;
+use App\Service\ComplementGenerator;
+use App\Service\SymfonyVersionProvider;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class UnsupportedBranchSubscriber implements EventSubscriberInterface
+{
+    private $symfonyVersionProvider;
+    private $issueApi;
+    private $complementGenerator;
+
+    public function __construct(SymfonyVersionProvider $symfonyVersionProvider, IssueApi $issueApi, ComplementGenerator $complementGenerator)
+    {
+        $this->symfonyVersionProvider = $symfonyVersionProvider;
+
+        $this->issueApi = $issueApi;
+        $this->complementGenerator = $complementGenerator;
+    }
+
+    /**
+     * Sets milestone on PRs that target non-default branch.
+     */
+    public function onPullRequest(GitHubEvent $event)
+    {
+        $data = $event->getData();
+        $repository = $event->getRepository();
```

Comment:
can be moved bellow where used, useless in case of return :)

Reply:
Thank you